packages/react-components/src/components/OnboardingChecklist/OnboardingChecklist.tsx

```diff
@@ -0,0 +1,184 @@
+import * as React from 'react';
+
+import { CheckCircle, ChevronDown } from '@livechat/design-system-icons';
+import cx from 'clsx';
+
+import { Button } from '../Button';
+import { Icon } from '../Icon';
+import { Heading, Text } from '../Typography';
+
+import { CheckListItem } from './components';
+import { IOnboardingChecklistProps } from './types';
+
+import styles from './OnboardingChecklist.module.scss';
+
+const baseClass = 'onboarding-checklist';
+const COMPLETE_CONTAINER_EIGHT = 96;
+
+export const OnboardingChecklist: React.FC<IOnboardingChecklistProps> = ({
+  activeId,
+  checkedId,
+  title,
+  titleLabel,
+  items,
+  onActiveChange,
+  placeholderClassName,
+  isCompleted = false,
+  completeItem,
+  className,
+}) => {
+  const [complete, setComplete] = React.useState(isCompleted);
```

Comment:
The name of this state can be confused with an external prop. Can you please rename is for better readability?